plasmapy/physics/transport.py

```diff
@@ -205,3 +209,1493 @@ def Coulomb_logarithm(T, n_e, particles, V=None):
     ln_Lambda = ln_Lambda.to(units.dimensionless_unscaled).value
 
     return ln_Lambda
+
+
+@check_quantity({"T_e": {"units": units.K, "can_be_negative": False},
+                 "n_e": {"units": units.m**-3},
+                 "T_i": {"units": units.K, "can_be_negative": False},
+                 "n_i": {"units": units.m**-3},
+                 })
+def classical_transport(coeffs, T_e, n_e, T_i, n_i, ion_particle, Z=None,
+                        B=0.0, model='Braginskii',
+                        field_orientation='parallel',
+                        coulomb_log_ei=None, V_ei=None,
+                        coulomb_log_ii=None, V_ii=None,
+                        hall_e=None, hall_i=None,
+                        mu=None, theta=None):
+    r"""
+    Braginskii-esque classical transport coefficients
+
+    Notes
+    -----
+    Classical transport theory is derived by using kinetic theory to close the
+    plasma two-fluid (electron and ion fluid) equations in the collisional
+    limit. This function uses fitting functions from literature to calculate
+    the transport coefficients, which are the resistivity, thermoelectric
+    conductivity, thermal conductivity, and viscosity, which can be used
+    to close the two-fluid equations.
+
+    Note well the assumptions in the derivation of classical transport:
+    (1) velocity distribution function is close to Maxwellian (no extremely
+        strong gradients) which is indicated when the following conditions
+        hold:
+    (1a) collisional frequency >> gyrofrequency
+    (1b) collisional mean free path << gradient scale length along field
+    (1c) gyroradius << gradient scale length perpendicular to field
+    (2) turbulent transport does not dominate
+
+    When classical transport is not valid, e.g. due to the presence of strong
+    gradients or turbulent transport, the transport is significantly increased
+    by these other effects. Thus, classical transport often serves as a lower
+    bound on the losses / transport encountered in a plasma.
+
+    Parameters
+    ----------
+    coeffs: list of string
+        Strings indicating the transport coefficients you want to calculate.
```

Comment:
I'm trying to think of a way to not have to pass this list here, see next comment...